docs-rst/installation/server-config/security.rst

```diff
@@ -29,27 +30,46 @@ You can use KSQL with a Kafka cluster in |ccloud|. For more information, see :re
 Configuring KSQL for Secured Confluent Schema Registry
 ------------------------------------------------------
 
-The following configuration connects KSQL with the Confluent Schema Registry over HTTPS.
+KSQL can be configured to connect to the Schema Registry over HTTP by setting the
+``ksql.schema.registry.url`` to the Schema Registry's HTTPS endpoint.
+Depending on your security setup, you may also need to supply a trust-store, (e.g. if the Schema
```

Comment:
`Depending on your security setup, you might also need to supply a trustStore and a keyStore. For example, a trustStore is required if the Schema Registry's SSL certificates are not trusted by the JVM by default; a keyStore is required if the Schema Registry requires mutual authentication.`

Reply:
done